autograder/question.py

```diff
@@ -85,8 +89,8 @@ def _internal_grade(self, helper, show_exceptions):
 
     def _score_helper(self, submission, additional_data = {}):
         """
-        Score the question, but make sure to return the result so
-        multiprocessing can properly pass it back.
+        Score the question, but make sure to return the result and stop grading
```

Comment:
You need to be more clear here.
This sentence makes it sound like this method stops the grading process always.